_test/nil4.go

```diff
@@ -0,0 +1,24 @@
+package main
```

Comment:
As _test/nil3.go and _test/nil4.go have exactly the same setup (only the main(), and the Output change), they could be merged together, right?